cpp/src/bindings.cpp

```diff
@@ -159,36 +162,44 @@ PYBIND11_MODULE(_pypowsybl, m) {
+                             py::object connectedComponentMode) {
+                 auto parameters = new load_flow_parameters();
+                 load_flow_parameters* config_params = pypowsybl::isConfigRead() ? pypowsybl::readParameters() : pypowsybl::defaultParameters();
+                 parameters->voltage_init_mode = voltageInitMode.is_none() ? config_params->voltage_init_mode : voltageInitMode.cast<pypowsybl::VoltageInitMode>();
+                 parameters->transformer_voltage_control_on = transformerVoltageControlOn.is_none() ? config_params->transformer_voltage_control_on : transformerVoltageControlOn.cast<bool>();
+                 parameters->no_generator_reactive_limits = noGeneratorReactiveLimits.is_none() ? config_params->no_generator_reactive_limits : noGeneratorReactiveLimits.cast<bool>();
+                 parameters->phase_shifter_regulation_on = phaseShifterRegulationOn.is_none() ? config_params->phase_shifter_regulation_on : phaseShifterRegulationOn.cast<bool>();
+                 parameters->twt_split_shunt_admittance = twtSplitShuntAdmittance.is_none() ? config_params->twt_split_shunt_admittance : twtSplitShuntAdmittance.cast<bool>();
+                 parameters->simul_shunt = simulShunt.is_none() ? config_params->simul_shunt : simulShunt.cast<bool>();
+                 parameters->read_slack_bus = readSlackBus.is_none() ? config_params->read_slack_bus : readSlackBus.cast<bool>();
+                 parameters->write_slack_bus = writeSlackBus.is_none() ? config_params->write_slack_bus : writeSlackBus.cast<bool>();
+                 parameters->distributed_slack = distributedSlack.is_none() ? config_params->distributed_slack : distributedSlack.cast<bool>();
+                 parameters->balance_type = balanceType.is_none() ? config_params->balance_type : balanceType.cast<pypowsybl::BalanceType>();
+                 parameters->dc_use_transformer_ratio = dcUseTransformerRatio.is_none() ? config_params->dc_use_transformer_ratio : dcUseTransformerRatio.cast<bool>();
+                 if (countriesToBalance.is_none()) {
+                     parameters->countries_to_balance = config_params->countries_to_balance;
+                     parameters->countries_to_balance_count = config_params->countries_to_balance_count;
+                 } else {
+                     std::vector<std::string> countries = countriesToBalance.cast<std::vector<std::string>>();
+                     parameters->countries_to_balance = pypowsybl::copyVectorStringToCharPtrPtr(countries);
+                     parameters->countries_to_balance_count = countries.size();
+                 }
+                 parameters->connected_component_mode = connectedComponentMode.is_none() ? config_params->connected_component_mode : connectedComponentMode.cast<pypowsybl::ConnectedComponentMode>();
+                 delete config_params;
```

Comment:
So here, as `config_params` has been allocated by java code, it must be deallocated in java code too.
It will cause issues if the content of pointers have not been **copied** to `parameters` : it's only an issue for the `char** countries_to_balance` field.